namespaces/cloud-platform-live-0.k8s.integration.dsd.io/court-probation-dev/resources/ecr.tf

```diff
@@ -25,3 +25,23 @@ resource "kubernetes_secret" "probation_services_ecr_credentials" {
     repo_url          = "${module.probation_services_ecr_credentials.repo_url}"
   }
 }
+
+module "ps_cps_pack_parser_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.1"
```

Comment:
ref=3.3 please, see https://github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials/releases

Reply:
Done

Reply:
sorry my mistake please set back to 2.1